core/src/apps/management/apply_settings.py

```diff
@@ -269,3 +277,18 @@ async def _require_confirm_experimental_features(
             reverse=True,
             br_code=BRT_PROTECT_CALL,
         )
+
+
+async def _require_confirm_hide_passphrase_from_host(
+    ctx: GenericContext, enable: bool
+) -> None:
+    if enable:
+        await confirm_action(
+            ctx,
+            "set_hide_passphrase_from_host",
+            "Hide passphrase",
+            "Only for development and beta testing!",
```

Comment:
it's not really. I think we can completely remove this line?

Reply:
Removed in [3759d83](https://github.com/trezor/trezor-firmware/pull/2829/commits/3759d83cf432023988b410f13fbcaf326dc875d0)